freqtrade/exchange/exchange.py

```diff
@@ -932,3 +935,22 @@ def timeframe_to_next_date(timeframe: str, date: datetime = None) -> datetime:
     new_timestamp = ccxt.Exchange.round_timeframe(timeframe, date.timestamp() * 1000,
                                                   ROUND_UP) // 1000
     return datetime.fromtimestamp(new_timestamp, tz=timezone.utc)
+
+
+def market_is_pair(market):
+    """
+    Return True if the market is a pair.
+    Currently pairs are defined as markets containing '/' character in its symbol.
+    """
+    return '/' in market.get('symbol', '')
+
+
+def market_is_active(market):
+    """
+    Return True if the market is active.
+    """
+    # "It's active, if the active flag isn't explicitly set to false. If it's missing or
+    # true then it's true. If it's undefined, then it's most likely true, but not 100% )"
+    # See https://github.com/ccxt/ccxt/issues/4874,
```

Comment:
🤣 😨

Reply:
Yeah... As an alternative, we could patch the 'active' field in the markets after ccxt to be it straight True/False as we need it...

Reply:
As another alternative, we can explicitly emphasize these pairs/exchanges with 'N/A' in the active column in the tabular output of this command instead of printing 'Yes'

Reply:
we should not change ccxt results i think ... 
this makes sense as implemented - my comment was related to the the 2nd line of the comment ... 

> If it's undefined, then it's most likely true, but not 100%

Reply:
I did not get though the difference there between "If it's missing" and "If it's undefined" 🤣